### PR TITLE
Onboarding flow updates

### PR DESCRIPTION
**Pull request by onurkenis**

### CODE REVIEW

**Review by barhantas — APPROVED** [review]

File: packages/web-ui/src/containers/Modal/ModalHeader/ModalHeader.tsx [hunk 1]
```diff
@@ -1,14 +1,21 @@
 import { Box } from "@material-ui/core";
 import React from "react";
 
+import { GovernanceTypography } from "@web-integration/components";
```
Comment (onurkenis) [inline packages/web-ui/src/containers/Modal/ModalHeader/ModalHeader.tsx:4]: import will be updated to @web-ui

File: packages/web-ui/src/flows/OnboardingFlow/screens/DepositFunds/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export { default } from "@web-integration/flows/OnboardingFlow/screens/DepositFunds/DepositFunds";
```
Comment (onurkenis) [inline packages/web-ui/src/flows/OnboardingFlow/screens/DepositFunds/index.ts:1]: same auto-import issue

File: packages/web-ui/src/implementations/app/HypernetWebUI.tsx [hunk 3]
```diff
@@ -256,11 +272,16 @@ export default class HypernetWebUI implements IHypernetWebUI {
     config: IRenderParams,
   ): Result<void, RenderError> {
     const renderReact = () => {
+      console.log("renderWalletConnectWidget");
```
Comment (onurkenis) [inline packages/web-ui/src/implementations/app/HypernetWebUI.tsx:275]: console.log will be removed

File: packages/web-ui/src/containers/Modal/ModalHeader/ModalHeader.tsx [hunk 4]
```diff
@@ -1,14 +1,21 @@
...
+
+import { useLayoutContext } from "@web-ui/contexts";
 import { useStyles } from "./ModalHeader.style";
```
Comment (onurkenis) [inline packages/web-ui/src/containers/Modal/ModalHeader/ModalHeader.tsx:7]: existing import by path will be replaced with alias

**Pull request merged**